Dockerfile

```diff
@@ -12,6 +12,11 @@ RUN go install tailscale.com/cmd/derper@main
 #Install Tailscale and Tailscaled
 RUN apk add tailscale --repository=https://dl-cdn.alpinelinux.org/alpine/edge/community
 
+FROM alpine:latest
+
+RUN mkdir -p /root/go/bin
+COPY --from=builder /root/go/bin/derper /root/go/bin/derper
+
```

Comment:
The container requires tailscale to be installed. Please add the following fix to copy over the binaries of tailscale and tailscaled as well as the required dependencies of the binaries.

Once the changes have been pushed I'll be happy to merge.

```
FROM alpine:latest

RUN mkdir -p /root/go/bin
COPY --from=builder /root/go/bin/derper /root/go/bin/derper
COPY --from=builder /usr/sbin/tailscaled /usr/sbin/tailscaled
COPY --from=builder /usr/bin/tailscale /usr/bin/tailscale

#Install Tailscale and requirements
RUN apk add curl iptables
```

Reply:
Good catch, I just moved the tailscale install down into the final container, that apk install there is reasonable rather than copying the binaries over.  The big reason for the builder pattern was to get rid of the go build dependencies.

Reply:
Got that moved down as well, good catch, I was thinking that was part of the build.